ws/templates/trips/__create_or_edit.html

```diff
@@ -120,6 +120,10 @@ <h2>Signup</h2>
 
   </div>
 
+  <div class="alert alert-warning">
+    If you have an approved budget for this trip, please remember to register this event on Atlas
+    <a href="https://forms.gle/dNrbKyjnWavEHmK3A">here</a>, which is required for reimbursement.
```

Comment:
"<click> here" should be avoided for accessibility & usability reasons

```suggestion
    If you have an approved budget for this trip, please remember to
     <a href="https://forms.gle/dNrbKyjnWavEHmK3A">register this event on Atlas</a>,
     which is required for reimbursement.
```

https://www.w3.org/QA/Tips/noClickHere